Modules/Microsoft365DSC/DSCResources/MSFT_MdcSubscriptionDefenderPlan/settings.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+    "resourceName": "MdcSubscriptionDefenderPlan",
+    "description": "Enables or disables Microsoft Defender plans for a subscription in Microsoft Defender for Cloud.",
+    "roles": {
+        "read": [],
+        "update": []
+    },
+    "permissions": {
+        "graph": {
+            "delegated": {
+                "read": [],
+                "update": []
+            },
```

Comment:
do you need to add relevnat permissions here?

Reply:
Since we are calling Azure APIs, we need to specify Azure required permissions, which is currently not supported